content/en/continuous_integration/pipelines/jenkins.md

```diff
@@ -648,23 +656,15 @@ Re/Initialize Datadog-Plugin Agent Http Client
 TRACE -> http://<HOST>:<TRACE_PORT>/v0.3/traces
 {{< /code-block >}}
 
-## Install the Datadog Agent
-
-The Datadog Jenkins plugin can either report metrics through the Datadog Agent or directly to Datadog if an API key is provided. If you don't have a Datadog Agent running on the Jenkins controller instance, Datadog recommends installing it first by following the [Agent installation instructions][14]. Whether you choose to use Agentless mode or the Agent-based mode, you are **required** to use the plugin.
+## Configure the Agent for log collection
```

Comment:
There is a duplicate here, see line 743

Reply:
I'm just confirming that we're comfortable removing this [Configure the Agent for log collection](https://docs.datadoghq.com/continuous_integration/pipelines/jenkins/?tab=reportthroughthedatadogagentrecommended#configure-the-agent-for-log-collection) section with instructions for Linux, Windows, and macOS.